src/autotype/AutoType.cpp

```diff
@@ -383,11 +385,24 @@ QList<AutoTypeAction*> AutoType::createActionFromTemplate(const QString& tmpl, c
         // some safety checks
         else if (tmplName.compare("delay",Qt::CaseInsensitive)==0) {
             if (num > 10000) {
-                return list;
+                QMessageBox::StandardButton reply;
+                reply = QMessageBox::question(0,"AutoType",
```

Comment:
You need to wrap your strings into tr() calls to make them translatable and please use spaces after commas.